src/dhtrunner.cpp

```diff
@@ -264,6 +263,13 @@ DhtRunner::run(const Config& config, Context&& context)
                 }
             }
         }
+        if (config.peer_discovery && config.peer_publish) {
```

Comment:
what if only `peer_discovery` or `peer_publish` is true ?

Reply:
It should be `config.peer_discovery or config.peer_publish` so i changed it to `if (peerDiscovery_)`.